js/client/modules/@arangodb/testsuites/endpoints.js

```diff
@@ -243,29 +251,30 @@ class endpointRunner extends tu.runInArangoshRunner {
       },
     };
     obj.orgArgs = _.clone(obj.instance.args);
+    let fatalError = false;
     return Object.keys(endpoints).reduce((results, endpointName) => {
       let testName = 'endpoint-' + endpointName;
       let testCase = endpoints[endpointName];
-
       if (obj.options.cluster || obj.options.skipEndpoints) {
         return {
           failed: 0,
           status: true,
           skipped: true
         };
       }
-      if (testCase.skip()) {
+      if (testCase.skip() || fatalError) {
         results[endpointName + '-' + 'all'] = {
           failed: 0,
           skipped: true,
           status: true,
-          message: 'test skipped'
+          message: fatalError? 'test skipped because of fatal error': 'test skipped'
```

Comment:
Is it possible to reach this code after `fatalError` is set to `true`?

Reply:
yes; its looping. the loop may not abort. 
Btw, this is just the backport.